aiqc/__init__.py

```diff
@@ -985,7 +1008,28 @@ def to_pandas(
 			columns:list = None, 
 			samples:list = None
 		):
-			return Dataset.Tabular.to_pandas(id, columns, samples)
+			df = Dataset.Tabular.to_pandas(id, columns, samples)
+			word_counts, feature_names = Dataset.Text.get_feature_matrix(df)
+			df = pd.DataFrame(word_counts.todense(), columns = feature_names)
+			return df
+
+		
+		def to_numpy(
+			id:int, 
+			columns:list = None, 
+			samples:list = None
+		):
+			df = Dataset.Tabular.to_pandas(id, columns, samples)
+			word_counts, feature_names = Dataset.Text.get_feature_matrix(df)
+			return word_counts, feature_names
```

Comment:
Where is the numpy array 🙃 ?

![image](https://user-images.githubusercontent.com/74990642/117289416-8ce3af00-ae3a-11eb-84f9-59cccd1e6735.png)

sparse matrices do not work with Keras

Reply:
need to do more research on this then, I was skeptical about the to_pandas() method too, with large datasets a numpy array was would be highly inefficient and probably even not fit in memory, will be looping back to this question as I work through the other issues on writing tests for the NLP module